recipes/mtcnn/meta.yaml

```
@@ -0,0 +1,49 @@
+{% set name = "mtcnn" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 56624f08e088e5cf062de8005df09696cdbc0f4c2fc3578483b19e0fb3984252
+
+build:
+  skip: True  # [py<34]
```

I think this recipe could be ``noarch``
```suggestion
  noarch: python
```